r5jxrm/r5jxrm/Program.cs

```diff
@@ -0,0 +1,1111 @@
+                return tet;
+            }
+        }
+
+        static public string[] Keveres(string[] pakli)
+        {
+            Random kevero = new Random();
+            int keveroindex;
+            int keveroindex2;
+            string temp;
+            for (int i = 0; i < 10000; i++)
+            {
+                keveroindex = kevero.Next(0, pakli.Length);
+                keveroindex2 = kevero.Next(0, pakli.Length);
+                temp = pakli[keveroindex];
+                pakli[keveroindex] = pakli[keveroindex2];
+                pakli[keveroindex2] = temp;
+            }
+            return pakli;
+        }
+
+
+        static public int checkHaVanAsz(string[] pakli)
+        {
+            bool containsAsz = Array.Exists(pakli, element => element == "A");
+            if (!containsAsz)
+            {
+                return 0;
+            }
+
+            int index = -1;
+            bool vegzett = false;
+            while (!vegzett)
+            {
+                index++;
+                if (pakli[index] == "A" || index == pakli.Length - 1)
+                {
+                    vegzett = true;
+                }
+            }
+            return index;
+        }
+
+        static private void tovabbLepesSzovegClear()
+        {
+            Console.WriteLine("Nyomj gombot a folytatáshoz!");
+            Console.ReadKey();
+            Console.Clear();
+        }
+
+        static private void tovabbLepesSzoveg()
+        {
+            Console.WriteLine("Nyomj gombot a folytatáshoz!");
+            Console.ReadKey();
+        }
+
+
+        static private int convertToInt(string[] pakli, int index)
+        {
+            int returnValue;
+            switch (pakli[index])
+            {
+                case "A":
+                    returnValue = 11;
+                    break;
+                case "K":
+                case "Q":
+                case "J":
+                case "10":
+                    returnValue = 10;
+                    break;
+                case "9":
+                    returnValue = 9;
+                    break;
+                case "8":
+                    returnValue = 8;
+                    break;
+                case "7":
+                    returnValue = 7;
+                    break;
+                case "6":
+                    returnValue = 6;
+                    break;
+                case "5":
+                    returnValue = 5;
+                    break;
+                case "4":
+                    returnValue = 4;
+                    break;
+                case "3":
+                    returnValue = 3;
+                    break;
+                case "2":
+                    returnValue = 2;
+                    break;
+                default:
+                    returnValue = 0;
+                    break;
+            }
+            return returnValue;
+        }
+        static void szinek(string beszin, List<ConsoleColor> szineklista)
```

Comment:
Lehetett volna csökkenteni a program méretét. Ez a függvény tipikusan jó példa, hogy mehetett volna pl egy külön static osztályba pl, vagy lehetett volna Extension method is akár.